code/modules/mob/living/silicon/robot/modules/module_medical.dm

```diff
@@ -139,15 +139,6 @@
 
 
 /obj/item/robot_module/medical/surgeon/respawn_consumable(mob/living/silicon/robot/R, amount)
-	var/obj/item/reagent_containers/syringe/S = locate() in equipment
-	if(S.mode == 2)
-		S.reagents.clear_reagents()
-		S.mode = initial(S.mode)
-		S.desc = initial(S.desc)
-		S.update_icon()
-	if(emag)
-		var/obj/item/reagent_containers/spray/PS = emag
-		PS.reagents.add_reagent(/datum/reagent/acid/polyacid, 2 * amount)
```

Comment:
this is supposed to be here, we moved this here from the crisis borg, towards the end of this file.

Reply:
The acid spray at least seems to be redundant with what the parent already does, but it relies on the weird emag var which no longer exists (and should really be renamed if it did, why is a ref to a spray called emag???).

The syringe, happy to restore but I don't understand what it's trying to do.

Reply:
that's fine for now, i'll take another look at why this was there in the first place and see about what should be added back in.